mmcv/ops/nms.py

```diff
@@ -302,3 +303,37 @@ def nms_match(dets, iou_threshold):
         return [dets.new_tensor(m, dtype=torch.long) for m in matched]
     else:
         return [np.array(m, dtype=np.int) for m in matched]
+
+
+def nms_rotated(boxes, iou_thr, labels=None, multi_label=False):
```

Comment:
It seems that the arguments passed into this function is a bit different with the common ones, e.g. `softnms` and `nms`. I wonder if it would be better to define a uniform style.